packages/file-types/zarr/src/ome-loaders/OmeZarrLoader.js

```diff
@@ -12,23 +21,47 @@ function hexToRgb(hex) {
 }
 
 export default class OmeZarrLoader extends AbstractTwoStepLoader {
+  constructor(dataSource, params) {
+    super(dataSource, params);
+    this.optionsSchema = imageOmeZarrSchema;
+  }
+
   async load() {
     const payload = await this.dataSource.getJson('.zattrs').catch(reason => Promise.resolve(reason));
     if (payload instanceof AbstractLoaderError) {
       return Promise.reject(payload);
     }
 
+    const { coordinateTransformations: coordinateTransformationsFromOptions } = this.options || {};
+
     const loader = await viv.loadOmeZarr(this.url, { fetchOptions: this.requestInit, type: 'multiscales' });
     const { metadata, data } = loader;
 
-    const { omero } = metadata;
+    const { omero, multiscales } = metadata;
 
     if (!omero) {
       console.error('Path for image not valid');
       return Promise.reject(payload);
     }
 
-    const { rdefs, channels } = omero;
+    if (!Array.isArray(multiscales) || multiscales.length === 0) {
+      console.error('Multiscales array must exist and have at least one element');
+    }
+    const { coordinateTransformations } = multiscales[0];
+
+    // Axes in v0.4 format.
+    const axes = getNgffAxes(multiscales[0].axes);
+
+    const transformMatrixFromOptions = coordinateTransformationsToMatrix(
+      coordinateTransformationsFromOptions, axes,
+    );
+    const transformMatrixFromFile = coordinateTransformationsToMatrix(
+      coordinateTransformations, axes,
+    );
+
+    const transformMatrix = transformMatrixFromFile.multiplyLeft(transformMatrixFromOptions);
```

Comment:
Maybe add a note that this allows for composing transformations from OMENGFF and user-provided

Reply:
Ah good point - I need to update the docs